icons/ktor.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ktor</title><path d="m15.979 23.982 3.983-3.984 4.017-4.016-4.224-4.224-3.776-3.776-.024 8.2zM8 16l3.983 3.983L16 24l4.224-4.224L24 16l-8.2-.024ZM8.035.018 4.052 4.001.035 8.018l4.224 4.224 3.776 3.776-.033-8.175zM16 8l-3.983-3.983L8 0 3.776 4.224 0 8l8.175-.034Z"/></svg>
```

Comment:
When using Ktor product icons from https://www.jetbrains.com/company/brand/#logos-and-icons-product-icons, I'm seeing some differences:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/6d0b6916-e061-4e38-85d5-fd3bc5b7aa8d)